freshtab/package.json

```diff
@@ -7,7 +7,10 @@
   },
   "devDependencies": {},
   "dependencies": {
+    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/ghostery-ios/master/latest.tgz",
```

Comment:
Use a specific commit, otherwise package-lock will complain once this link is updated to point to a new version.